libs/components/tabs/src/lib/modules/vertical-tabset/vertical-tab.component.scss

```diff
@@ -1,35 +1,184 @@
+    var(--sky-space-inset-pillarbox-2_3-top-m)
+      var(--sky-space-inset-pillarbox-2_3-right-m)
+      var(--sky-space-inset-pillarbox-2_3-bottom-m)
+      var(--sky-space-inset-pillarbox-2_3-left-m)
+  );
+  margin: var(
+    --sky-override-vertical-tab-margin,
+    0 0 0 var(--sky-space-inline-s)
+  );
   display: flex;
   text-decoration: none;
+  color: var(
+    --sky-override-vertical-tab-font-color,
+    var(--sky-color-text-deemphasized)
+  );
+  border-radius: var(--sky-border-radius-0);
 
-  &:not(.sky-deemphasized) {
-    color: $sky-text-color-default;
+  &:hover {
+    color: var(
+      --sky-override-vertical-tab-hover-font-color,
+      var(--sky-color-text-default)
+    );
+    background-color: var(
+      --sky-override-vertical-tab-background-color-hover,
+      var(--sky-color-background-nav-hover)
+    );
+
+    // TODO: Tokenize with sectioned form
+    .sky-vertical-tab-right-arrow {
+      color: darken($sky-text-color-icon-borderless, 20%);
+    }
   }
 
-  &:hover {
-    background-color: $sky-background-color-neutral-light;
+  &:active {
+    background-color: var(
+      --sky-override-vertical-tab-background-color-active,
+      var(--sky-color-background-nav-active)
+    );
+    border-left: var(
+      --sky-override-vertical-tab-border-left-active,
+      var(--sky-border-width-selected-l) solid var(--sky-color-border-selected)
+    );
+    color: var(
+      --sky-override-vertical-tab-active-font-color,
+      var(--sky-color-text-default)
+    );
+    padding-left: var(
+      --sky-override-vertical-tab-padding-left-active,
+      calc(
+        var(--sky-space-inset-pillarbox-2_3-left-m) - var(
+            --sky-border-width-selected-l
+          )
+      )
+    );
   }
 
-  &:focus-visible {
-    @include mixins.sky-focus-outline;
+  &:not(:active):focus-visible {
+    background-color: var(
+      --sky-override-vertical-tab-background-color-focus,
+      var(--sky-color-background-nav-focus)
+    );
+    outline: var(
+      --sky-override-vertical-tab-outline-focus,
+      var(--sky-border-width-selected-m) solid var(--sky-color-border-selected)
+    );
+    outline-offset: var(
+      --sky-override-vertical-tab-outline-focus-offset,
+      calc(var(--sky-border-width-selected-m) * -1)
+    );
+    box-shadow: var(
+      --sky-override-vertical-tab-box-shadow-focus,
+      var(--sky-elevation-focus)
+    );
   }
 
-  &:hover .sky-vertical-tab-right-arrow {
-    color: darken($sky-text-color-icon-borderless, 20%);
+  &:not(.sky-vertical-tab-active):not(:active):hover {
+    padding-left: var(
+      --sky-override-vertical-tab-padding-left-hover,
+      calc(
+        var(--sky-space-inset-pillarbox-2_3-left-m) - var(
+            --sky-border-width-selected-s
+          )
+      )
+    );
+    border-left: var(
+      --sky-override-vertical-tab-border-left-hover,
+      var(--sky-border-width-selected-s) solid var(--sky-color-border-selected)
+    );
   }
 }
 
 .sky-vertical-tab-active {
```

Comment:
Just for clarity - the `sky-vertical-tab-active` is the "selected" class and not the browser active state. I did not modify the existing class here for backwards compatibility.